code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -209,7 +209,7 @@
 	accuracy = 1
 	matter = list(DEFAULT_WALL_MATERIAL = 200)
 	w_class = ITEMSIZE_NORMAL
-	amount_per_transfer_from_this = 20
+	amount_per_transfer_from_this = 120
 	possible_transfer_amounts = list(10,20,30,60,120)
```

Comment:
Should we just set it to the max of `possible_transfer_amounts` in `Initialize()`, instead?

Reply:
I was thinking about that but I'm not sure if I want to override initialize for this.